analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -108,6 +117,10 @@ protected override void Initialize(SonarAnalysisContext context)
         protected override void AnalyzeSonar(SyntaxNodeAnalysisContext context, bool isTestProject, bool isScannerRun, SyntaxNode body, ISymbol symbol)
         {
             var enabledAnalyzers = SonarRules.Where(x => x.SupportedDiagnostics.Any(descriptor => IsEnabled(context, isTestProject, isScannerRun, descriptor))).ToArray();
+            if (!Configuration.ForceSonarCfg)
+            {
+                enabledAnalyzers = enabledAnalyzers.Where(x => !x.SupportedDiagnostics.Any(descriptor => AllRules.Keys.Any(dd => descriptor.Id == dd.Id))).ToArray();
```

Comment:
This is tricky, because `Analyze_Severity_ExecutesWhenAll` now works by funny coincidence. But as this should go away in the next PR, it's not an issue (yet).